docs/docs/APIdocs/layers/optim/Adadelta.md

```diff
@@ -0,0 +1,34 @@
+## Adadelta ##
+
+
+*AdaDelta* implementation for *SGD* 
+It has been proposed in `ADADELTA: An Adaptive Learning Rate Method`.
+http://arxiv.org/abs/1212.5701.
+
+**Scala:**
+```scala
+Adadelta(decayRate: Double = 0.9, Epsilon: Double = 1e-10)
+```
+**Python:**
+```python
+AdaDelta(decayrate = 0.9, epsilon = 1e-10, bigdl_type="float")
+```
+
+
+**Scala example:**
+```scala
+optimizer.setOptimMethod(new Adadelta(0.9, 1e-10))
```

Comment:
But we would not expect user to use `forward` and `backward` either, while we still put `forward`/`backward` in each layer examples. I thought the examples are to explain the behavior of the module.